sway-core/tests/sway_to_ir/mutable_struct.ir

```diff
@@ -0,0 +1,31 @@
+script {
```

Comment:
Toby moved all IR tests to https://github.com/FuelLabs/sway/tree/master/test/src/ir_generation and changed them to use `FileCheck`.